CONTRIBUTING.md

```diff
@@ -350,6 +350,30 @@ Notice how we do all of these things manually:
 
 We have full access to the [Cypress API](https://docs.cypress.io/api) so any additional interactions or custom logic can be easily added.
 
+### Accessibility testing
+
+We use an automated script to evaluate each example for accessibility violations using [`cypress-axe`](https://github.com/component-driven/cypress-axe). The setup for this can be found in the `a11y` workspace.
```

Comment:
one more suggestion

```suggestion
We use an automated script to evaluate each component example for accessibility violations using [`cypress-axe`](https://github.com/component-driven/cypress-axe). The setup for this can be found in the `a11y` workspace, and the component examples can be found in the `examples` workspace.
```